app/assets/prototype/prototype.scss

```diff
@@ -0,0 +1,15 @@
+/* Prototype */
+
+// NHSUK Includes
+@import 'node_modules/nhsuk-frontend/packages/core/settings/all';
+@import 'node_modules/nhsuk-frontend/packages/core/tools/all';
+
+// Utils
+@import '../hee/utils/mixins';
+
+// Template
+@import 'template/layout';
+@import 'template/example';
```

Comment:
@dawntaroni getting an error here:

```
Error in plugin "sass"
Message:
    app/assets/prototype/prototype.scss
Error: File to import not found or unreadable: template/example.
        on line 12 of app/assets/prototype/prototype.scss
>> @import 'template/example';
Error in plugin "sass"
Message:
    app/assets/prototype/prototype.scss
Error: File to import not found or unreadable: template/example.
        on line 12 of app/assets/prototype/prototype.scss
>> @import 'template/example';
```

Reply:
Seeing an issue with the example layouts which is probably related:

![image](https://user-images.githubusercontent.com/8265949/201698040-0000821a-a4ac-4bd9-8c1e-a29b4be1e9e2.png)